### PR TITLE
2. Tech page - Past and Future

### PR DESCRIPTION
**Pull request by Xanndarr**
### Motivation

- [Related story](https://trello.com/c/HlKb0Jko/27-tech-page%3A-past-%26-future-slice)

Blocked: We need formal decision on posts to tag as `tried-and-tested` and `growing-trends`. At the moment I've just tagged random ones.

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by gregdorward** [comment]
@Xanndarr in its final state, will this solution allow us to pull through different blogs to those shown on the homepage? As in, could we display blogs a, b and c on the Homepage, and x, y and z in the Tried and Tested slice?

File: site/compiler/people.test.js [hunk 1]
```diff
@@ -365,9 +365,9 @@ describe('site/compiler', () => {
         createStateNavigator(),
       );
 
-      expect(routes.length).to.equal(14);
-      expect(routes[12].filePath).to.equal('about-us/people/alex/index.html');
-      expect(routes[13].filePath).to.equal('about-us/people/sari/index.html');
+      expect(routes.length).to.equal(15);
+      expect(routes[13].filePath).to.equal('about-us/people/alex/index.html');
+      expect(routes[14].filePath).to.equal('about-us/people/sari/index.html');
```
Comment (JoeStanton) [inline site/compiler/people.test.js]: All of these specific indices are getting quite cumbersome. Time for some sorting/filtering in the expectations I think.

Comment (JoeStanton) [inline site/compiler/people.test.js]: All of these specific indices are getting quite cumbersome. Time for some sorting/filtering in the expectations I think. Separate PR though

File: state/index.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import fetch from 'node-fetch';
 import Promise from 'bluebird';
-import { getFeaturedPosts } from '../site/fetchers/featured-blog-posts';
+import { getBlogPosts } from '../site/fetchers/blog-posts';
```
Comment (JoeStanton) [inline state/index.js]: `getTaggedPosts` ?

File: site/pages/technology/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          <img className={styles.techRoundTable} src={techRoundTableImage} alt="tech roundtable" />
+        </div>
+      </section>
+    </section>
+    <section className={styles.pastAndFuture}>
+      <div className={styles.leftBlogs}>
+        <BlogSlice blogPosts={[]} title={'Tried and tested'} />
+      </div>
+      <div className={styles.rightBlogs}>
+        <BlogSlice blogPosts={[]} title={'Growing Trends'} />
```
Comment (JoeStanton) [inline site/pages/technology/index.js]: don't think you want/need those braces as the string is a literal?

File: site/pages/technology/blog-slice/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import React from 'react';
+import styles from './style.css';
+import BlogEntry from '../../../components/blog-entry';
+import type { BlogPost } from '../../../components/blog-entry';
+
+const blogSlice = ({ blogPosts, title }: { blogPosts: Array<BlogPost>, title: string }) => {
```
Comment (JoeStanton) [inline site/pages/technology/blog-slice/index.js]: In JSX, lower-case tag names (eg. `<blogPost>`) are considered to be HTML tags. Maybe we should always uppercase these names (it doesn't really matter in this instance as the export is default but still think its better to be consistent).

File: site/compiler/people.test.js (hunk 1 above)
Comment (Xanndarr) [inline site/compiler/people.test.js]: Gonna try pushing MVP out first, then hopefully if I get time, i'll rework some of these horrible tests!

File: site/css/_divider.css [hunk 5]
```diff
@@ -6,9 +6,10 @@
 }
 
 .divider:before {
+  padding-left: 2px;
   content:"";
   position: absolute;
-  right: -6px;
+  right: -4px;
```
Comment (Xanndarr) [inline site/css/_divider.css]: Doesn't affect any other dividers since we only use full width divs. This only lines up the divider with a half width div.

**Comment by gregdorward** [comment]
@Xanndarr, I'll add to this list as I find things that need a bit of tweaking.

- [ ] Make "Tried and tested" and "Growing trends" headings left aligned, as per the designs

- [ ] Change font of "Growing Trends" to "Growing trends" (lower case t)

**Pull request merged**